package.json

```diff
@@ -35,7 +35,7 @@
     "@cosmjs/proto-signing": "0.25.3",
     "@cosmjs/stargate": "0.25.3",
     "@cosmjs/stream": "0.25.3",
-    "@cosmjs/tendermint-rpc": "0.25.3",
+    "@cosmjs/tendermint-rpc": "https://gitpkg.now.sh/desmos-labs/cosmjs/packages/tendermint-rpc?riccardo/add-block-search",
```

Comment:
Wow, very nice. I never knew how to import non-tagged releases from lerna monorepos

Reply:
Nice. But where is the TS -> JS compiled?

Reply:
@webmaster128 I think you can read more [here](https://gitpkg.vercel.app/guide/#advanced-how-this-function-is-implemented). It should be compiled by them